server/exercise/src/main/scala/com/eigengo/lift/exercise/MultiPacket.scala

```diff
@@ -4,10 +4,10 @@ import scodec.bits.BitVector
 
 case class PacketWithLocation(sourceLocation: SensorDataSourceLocation, payload: BitVector)
 
-case class MultiPacket(packets: List[PacketWithLocation]) {
+case class MultiPacket(timestamp: Long, packets: List[PacketWithLocation]) {
```

Comment:
Sorry, just read the last packet in the PR notes - so scrubbed the previous comment!